test/manual/amp-analytics-visible-reset.amp.html

```diff
@@ -0,0 +1,144 @@
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+
+</head>
+
+<body>
+  <amp-analytics>
+    <script type="application/json">
+      {
+        "transport": {"beacon": false, "xhrpost": false},
+        "requests": {
+          "base": "fake.com/ad1?reset=3000"
+        },
+        "triggers": {
+          "reset": {
+            "on": "visible",
+            "request": "base",
+            "visibilitySpec": {
+              "selector": "#ad1",
+              "visiblePercentageMin": "0",
+              "reset": 3000
+            }
+          }
+        }
+      }
+    </script>
+  </amp-analytics>
+  <amp-analytics>
+    <script type="application/json">
+      {
+        "transport": {"beacon": false, "xhrpost": false},
+        "requests": {
+          "base": "fake.com/ad1?reset=200&continoustimemin=2000"
+        },
+        "triggers": {
+          "reset": {
+            "on": "visible",
+            "request": "base",
+            "visibilitySpec": {
+              "selector": "#ad1",
+              "visiblePercentageMin": "0",
+              "continuousTimeMin": 2000,
+              "reset": 200
+            }
+          }
+        }
+      }
+    </script>
+  </amp-analytics>
+  <amp-analytics>
+    <script type="application/json">
+      {
+        "transport": {"beacon": false, "xhrpost": false},
+        "requests": {
+          "base": "fake.com/ad1?reset=100"
+        },
+        "triggers": {
+          "reset": {
+            "on": "visible",
+            "request": "base",
+            "visibilitySpec": {
+              "selector": "#ad1",
+              "visiblePercentageMin": "0",
+              "reset": 100
+            }
+          }
+        }
+      }
+    </script>
+  </amp-analytics>
+  <amp-analytics>
+    <script type="application/json">
+      {
+        "transport": {"beacon": false, "xhrpost": false},
+        "requests": {
+          "base": "fake.com/ad2?reset=true"
+        },
+        "triggers": {
+          "reset": {
+            "on": "visible",
+            "request": "base",
+            "visibilitySpec": {
+              "selector": "#ad2",
+              "visiblePercentageMin": "0",
+              "reset": true
+            }
+          }
+        }
+      }
+    </script>
+  </amp-analytics>
+  <h2>fake3pAd send render-start w/ width & height</h2>
```

Comment:
Nit: Replacing ampersand char with ampersand HTML entity would be more correct, but it's a test so not too concerned :)

Reply:
removed : )